packages/api-contract/src/MetaRegistry.spec.ts

```diff
@@ -17,8 +17,6 @@ function compare (meta: MetaRegistry, other: any): void {
   try {
     expect(meta.typeDefs).toEqual(other);
   } catch (error) {
-    console.error(JSON.stringify(meta.typeDefs));
```

Comment:
We actually do want this - so in the case of stuff going wrong, we can actually see the expected output - and can actually easily adjust as needed.

Reply:
PS And it will restore the linting.